src/pages/en/guides/styling.md

```diff
@@ -278,6 +273,8 @@ As an alternative to `src/styles/global.css`, You may also add Tailwind utilitie
 
 As of [version 0.20.0](https://github.com/withastro/astro/releases/tag/astro%400.20.0), Astro will no longer bundle, build and process `public/` files. Previously, we'd recommended putting your tailwind files in the `public/` directory. If you started a project with this pattern, you should move any Tailwind styles into the `src` directory and import them in your template using [Astro.resolve()][astro-resolve]:
 
+As of [version 0.24.0-next](https://github.com/withastro/astro/releases/tag/astro%400.24.0-next.0) Astro no longer supports [Astro.resolve()][astro-resolve]. See the [Migration Guide](migration-guide) for more details.
```

Comment:
Can someone make sure that the migration guide link is correct? I'm not sure how these are generated, so I just made a guess here: `[Migration Guide](migration-guide)`

Reply:
Ah yes, tested on https://deploy-preview-193--astro-docs-2.netlify.app/en/guides/styling/ and it seems to be broken. How do I fix it?

Reply:
Ok, using `/en/migrate` for now, not sure that's the right way to do it or not.